code/datums/mutations/void_magnet.dm

```diff
@@ -41,3 +37,42 @@
 /datum/action/cooldown/spell/void/cast(atom/cast_on)
 	. = ..()
 	new /obj/effect/immortality_talisman/void(get_turf(cast_on), cast_on)
+
+/// The cursed "void invocation" action, that has a chance of casting itself on its owner randomly on life ticks.
+/datum/action/cooldown/spell/void/cursed
+	name = "Convoke Void" //magic the gathering joke here
+	desc = "A rare genome that attracts odd forces not usually observed. May sometimes pull you in randomly."
+	/// A multiplier applied to the probability of the curse appearing every life tick
+	var/curse_probability_modifier = 1
+
+/datum/action/cooldown/spell/void/cursed/Grant(mob/grant_to)
+	. = ..()
+	if(!owner)
+		return
+
+	RegisterSignal(grant_to, COMSIG_LIVING_LIFE, .proc/on_life)
```

Comment:
I disagree with the new theme of registering everything to ``COMSIG_LIVING_LIFE``.

``/mob/living/carbon/human/Life()`` calls every mutation's ``on_life``, I'd rather we be consistent with other mutations (for now, anyway) and use that instead.

Reply:
Just hoping that all mutations are moved to this new system, then.